packages/eslint-plugin-orbit-components/__tests__/useRtl.test.ts

```diff
@@ -0,0 +1,187 @@
+import ruleTester from "../ruleTester";
+import useRtl, { ERROR_RTL_SPACING, ERROR_BORDER_RADIUS, errorBasic } from "../src/rules/useRtl";
+
+describe("use-rtl", () => {
+  ruleTester.run("use-rtl", useRtl, {
+    valid: [
+      {
+        code: `
+          import styled from 'styled-components';
+          import { right } from "@kiwicom/orbit-components/lib/utils/rtl";
+
+          const StyledWrapper = styled.div\`
+              \${right}: 0;
+            \`
+          `,
+      },
+      {
+        code: `
+          import styled from 'styled-components';
+          import { left } from "@kiwicom/orbit-components/lib/utils/rtl";
+
+          const StyledWrapper = styled.div\`
+              margin-\${left}: 0;
+            \`
+          `,
+      },
+      {
+        code: `
+          import styled from 'styled-components';
+          import { left } from "@kiwicom/orbit-components/lib/utils/rtl";
+
+          const StyledWrapper = styled.div\`
+              margin-\${left}: 0;
+            \`
+          `,
+      },
+      {
+        code: `
+          import styled from 'styled-components';
+
+          const StyledWrapper = styled.div\`
+              margin: 10px 15px 0 15px;
+            \`
+          `,
+      },
+      {
+        code: `
+          import styled from 'styled-components';
+
+          const StyledWrapper = styled.div\`
+              margin: 10px 15px 3px;
+            \`
+          `,
+      },
+      {
+        code: `
+          import styled from 'styled-components';
+
+          const StyledWrapper = styled.div\`
+              border-radius: 3px 3px 5px 5px;
+            \`
+          `,
+      },
+      {
+        code: `
+          import styled from 'styled-components';
+
+          const StyledWrapper = styled.div\`
+              border-radius: 3px 3px 5px 5px;
```

Comment:
nope, it's correct code, it's under `valid` and it's for case when you do not need `borderRadius` function. It has `3px 3px` on the top, so when RTL will be applied - it will be the same.